### PR TITLE
Add .NET 5 target to integration tests

### PR DESCRIPTION
**Pull request by kevingosse**
Summary of the changes:

- Edited the major version of integrations to target 5.x assemblies
- Added a job to run .NET 5 integration tests
- Added the .NET 5 SDK to all impacted pipelines
- Updated the framework detection logic in tests because the TFM does not contain "netcoreapp"
- Updated MySQL in Alpine because of weird failures with Alpine 3.12 (the .NET 5 image is not available for Alpine 3.10)

### CODE REVIEW

**Review by zacharycmontoya — APPROVED** [review]
LGTM

**Review by lucaspimentel — APPROVED** [review]
🎉

File: test/Directory.Build.props [hunk 1]
```diff
@@ -2,8 +2,8 @@
   <Import Project="..\Directory.Build.props" />
 
   <PropertyGroup>
-    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
-    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netcoreapp2.1;netcoreapp3.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net452;net461;netcoreapp2.1;netcoreapp3.0;netcoreapp3.1;net5.0</TargetFrameworks>
```
Comment (zacharycmontoya) [inline test/Directory.Build.props:5]: Since net5.0 is not GA yet, should we include `net5.0` based on a build property?

**Comment by tonyredondo** [comment]
I don’t know if it should part of this PR but the CLI tool and standalone projects should be updated to net5.0 as well.

File: test/Directory.Build.props (hunk 1 above)
Comment (kevingosse) [reply test/Directory.Build.props:5]: That's a very good point. I did the whole thing with VS Preview, and forgot it wouldn't work with the "normal" version

Comment (kevingosse) [reply test/Directory.Build.props]: Added conditions. Now net5.0 should only be enabled when using VS Preview.

**Comment by kevingosse** [comment]
> I don’t know if it should part of this PR but the CLI tool and standalone projects should be updated to net5.0 as well.

Should they? The goal is to make sure that our product works with .NET 5.0, not to actually use .NET 5.0 🤔

**Pull request merged**